apps/theme/layouts/forms/DefaultFormLayout/fields/Checkbox.tsx

```diff
@@ -99,6 +119,34 @@ export const CheckboxField: React.FC<CheckboxProps> = ({ field }) => {
                     <label htmlFor={"checkbox-" + fieldId + option.value}>{option.label}</label>
                 </CheckboxGroup>
             ))}
+            {field.settings.otherOption && (
+                <CheckboxGroup>
+                    <CheckboxButton
+                        name={fieldId}
+                        type="checkbox"
+                        id={"checkbox-" + fieldId + otherOption.value}
+                        checked={checked({ option: otherOption, value })}
+                        onChange={e => {
+                            change({ option: otherOption, value, onChange });
+                            if (e.target.checked && otherInputRef.current) {
+                                otherInputRef.current.disabled = false;
+                                otherInputRef.current.focus();
```

Comment:
Would the following eliminate all these lines of code?
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

Let's try that, but I believe my previous comment would also need to be applied in order for the `autofocus` to work.